### PR TITLE
Fixtures: fix French translation of cap

### PR DESCRIPTION
**Pull request by dunglas**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | yes
| New feature?    | yes
| BC breaks?      | yes
| Deprecations?   | no 
| Related tickets | n/a
| License         | MIT

image

This isn't a "casquette", it's a "bonnet" 😅

### CODE REVIEW

**Comment by Pierstoval** [comment]
You sure? 😅 
image

**Comment by dunglas** [comment]
According to the product’s picture it’s definitely a « bonnet », even if there is only one word for « casquette » and « bonnet » in English.

**Comment by Pierstoval** [comment]
Oh, I didn't understand why you were talking about pictures, but the picture didn't actually show on my browser 😅 I opened it with another browser and I saw it.

Totally okay then 👍

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/cap.yaml [hunk 1]
```diff
@@ -25,16 +25,16 @@ sylius_fixtures:
...
                                             fr_FR:
-                                                name: 'Casquettes'
+                                                name: 'Bonnets'
                                         children:
                                             -   code: 'simple_caps'
                                                 translations:
                                                     en_US:
                                                         name: 'Simple'
-                                                        slug: 'caps/simple'
+                                                        slug: 'bonnets/simple'
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/cap.yaml]: We should probably leave the slug for `en_US` translation

**Comment by romainricard** [comment]
Seconded, this is a _bonnet_ in French.

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/cap.yaml (hunk 1 above)
Comment (jacquesbh) [reply src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/cap.yaml]: I agree.

Comment (dunglas) [reply src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/cap.yaml]: Oops, it was a mistake. Fixed!

**Comment by Zales0123** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "patch-1"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by pamil** [comment]
Thanks, Kévin! :tada:

**Pull request merged**